addOns/authhelper/src/main/javahelp/org/zaproxy/addon/authhelper/resources/help/contents/session-header.html

```diff
@@ -57,12 +57,24 @@ <H2>Automation Framework</H2>
           Authorization: "Bearer: {%json:AuthenticationResult.AccessToken%}"
 </pre>
 
-<p>
+<H2>API</H2>
 
+Header Based Session Management can be configured via ZAP's API, using the following parameters:
+<pre>
+    contextId
+    methodName
+    methodConfigParams
+</pre>
+For example:
+<table border="1">
+  <tr><td>contextId</td><td>1</td></tr>
+  <tr><td>methodName</td><td>headerBasedSessionManagement</td></tr>
+  <tr><td>methodConfigParams</td><td>headers=Authorization:Bearer 8c6976e5b5410415bde908bd4dee15dfb167a9c873fc4bb8a81f6f2ab448a918</td></tr>
```

Comment:
Worth including a second header? So people know what the separator is.

Reply:
Sure

Reply:
done

Reply:
Just adding "\n" will be misleading, it's a newline not the two characters.

Reply:
Just add a note underneath then?

> To add multiple headers specify each header name and value pair new line delimited.

Or something like that.

Reply:
wfm